SR2024/2023-09-27-chase-details-for-kits.md

```diff
@@ -0,0 +1,48 @@
+---
+to: SR2024 teams who have not confirmed any details
+subject: "Action Required: Details needed to receive your kit for Student Robotics 2024"
+attachments:
+  - Kit Disclaimer (PDF): https://drive.google.com/file/d/1a4Uh3O5qYElmA_QPztpVyvAByUGf4T3H/view
+---
+
+Hello,
+
+We hope you're as excited about the start of the Student Robotics year as we
+are!
+
+As you're hopefully aware, the year starts with a virtual [Kickstart][kickstart]
+event on Saturday 21st October which gives competitors and team supervisors an
+introduction to this year's competition and overview of how to build and
+engineer their robot.
+
+Ahead of Kickstart we'd planning to send out your loaned [robotics kit][kit-docs],
+for your team to use as the core of your robot. As mentioned in our previous
+email confirming your place, we need some details for you before we can do that.
+So that your kit can arrive by Kickstart please complete the following by this
+**Sunday 1st October**:
+
+- A _secondary contact_ for your team. This should be someone at your
+  institution (e.g: head of department). They should know about the
+  competition, and be able to act as a backup contact.
+- You need to agree to our _kit disclaimer_ (PDF attached). This disclaimer
+  covers the terms and conditions under which you may use the kit and outlines
+  the limits of our responsibilities. Please sign it, scan it and email it back
+  to us.
+- Confirmation of your _shipping address_.
+
+Please fill in this form to confirm your shipping address and provide details of
+your secondary contact:
+
+    https://forms.gle/jgJ6fhspWUNyWR5X9
+
+The shipping address we have for you is:
+
+    SHIPPING_ADDRESS
+
+To ensure that you receive your robotics kit before Kickstart, please provide
+us with the details necessary by Sunday 1st October.
```

Comment:
Question: Is it intentional that the date is duplicated here? (I assume yes for emphasis)

Reply:
Yeah, it was duplicated in the place confirmation email too.